fhir-bucket/src/main/java/com/ibm/fhir/bucket/app/Main.java

```diff
@@ -579,7 +699,18 @@ protected void shutdown() {
         this.resourceHandler.signalStop();
         
         
-        this.scanner.waitForStop();
+        if (this.scanner != null) {
+            this.scanner.waitForStop();
+        }
+        
+        if (driveReindexOperation != null) {
+            driveReindexOperation.waitForStop();
+        }
+        
+        if (cmsPayerWorkload != null) {
+            cmsPayerWorkload.waitForStop();
```

Comment:
I suggest not using cmspayer.  let's just call it workload

Reply:
done